script.js

```diff
@@ -2,49 +2,73 @@
 //Declare a variable called 'name' that has the value of your name.
 
 /*code here*/
-
+var name="Jose";
 //#2
 //create an if statement that checks to see if your name is equal to 'Ben'
 //If yes, log 'yes'
 //otherwise log 'no'
 //note: use console.log to log the string
 
 /*code here*/
-
+if( name==='Ben')
+  Console.log("yes");
+else 
+  Console.log("no");
```

Comment:
Generally in JS we use single quotes around strings.  If you're coming from a language like Java then you might be used to always having to use double quotes.  In JS though this pattern is generally preferred.  `console.log('no');`

Also, don't capitalize `Console`.  It should be `console.log`